spec.md

```diff
@@ -809,6 +1009,7 @@ The following HTTP Headers are defined for this operation:
 | organization_guid* | string | Deprecated in favor of `context`. The Platform GUID for the organization under which the Service Instance is to be provisioned. Although most Service Brokers will not use this field, it might be helpful for executing operations on a user's behalf. MUST be a non-empty string. |
 | space_guid* | string | Deprecated in favor of `context`. The identifier for the project space within the Platform organization. Although most Service Brokers will not use this field, it might be helpful for executing operations on a user's behalf. MUST be a non-empty string. |
 | parameters | object | Configuration options for the Service Instance. Service Brokers SHOULD ensure that the client has provided valid configuration parameters and values for the operation. |
+| alias | string | Alias for this instance. |
```

Comment:
I think alias is suppose to be used like instance name. and I am assuming the platform allows this detail and this is a way to get the platforms name of the instance into the broker's context.

Why not call it name? or instance_name... Or leave it out and always operate on service_id, and have the platform deal with the translation.